.github/workflows/ci.yaml

```
@@ -0,0 +1,286 @@
+                  -v "${vernum}" \
+                  mkdir -p "${CONTAINER_IMAGE_ROOT}"
+          ./run_sdk_container -n "${image_container}" -C "${packages_image}" \
+                  -v "${vernum}" \
+                  ./set_official --board="${arch}-usr" "${official_arg}"
+          ./run_sdk_container -n "${image_container}" -C "${packages_image}" \
+                  -v "${vernum}" \
+                  ./build_image --board="${arch}-usr" --group="${channel}" \
+                                --output_root="${CONTAINER_IMAGE_ROOT}" \
+                                --torcx_root="${CONTAINER_TORCX_ROOT}" prodtar container
+
+          # Copy logs
+          ./run_sdk_container -n "${image_container}" -C "${packages_image}" -v "${vernum}" \
+              tar -cJf ebuild_logs.tar.xz /build/${arch}-usr/var/log/portage \
+              /build/${arch}-usr/var/tmp/portage
+
+          docker commit "${image_container}" "${image}:${docker_vernum}"
+          docker rm -f "${image_container}"
+
+      - name: Build VM image
+        shell: bash
+        run: |
+          set -euo pipefail
+          set +x
+
+          source ci-automation/ci_automation_common.sh
+
+          vms_container="flatcar-vms-${docker_vernum}"
+          images_out="images/"
+
+          has_packet=0
+          has_pxe=0
+          formats="${IMAGE_FORMATS}"
+          for format in "${formats}";do
+            [[ "${format}" = 'packet' ]] || [[ "${format}" = 'equinix_metal' ]] && has_packet=1
+            [[ "${format}" = 'pxe' ]] && has_pxe=1
+          done
+
+          [[ ${has_packet} -eq 1 ]] && [[ ${has_pxe} -eq 0 ]] && set -- 'pxe' "${@}"
+          if echo "$formats" | tr ' ' '\n' | grep -q '^vmware'; then
+            formats=$(echo "$formats" | tr ' ' '\n' | sed '/vmware.*/d')
+            formats+=" vmware vmware_insecure vmware_ova vmware_raw"
+          fi
+          if echo "$formats" | tr ' ' '\n' | grep -q -P '^(ami|aws)'; then
+            formats=$(echo "$formats" | tr ' ' '\n' | sed '/ami.*/d' | sed '/aws/d')
+            formats+=" ami ami_vmdk"
+          fi
+          # Keep compatibility with SDK scripts where "equinix_metal" remains unknown.
+          formats=$(echo "$formats" | tr ' ' '\n' | sed 's/equinix_metal/packet/g')
+
+          for format in ${formats}; do
+              echo " ###################  VENDOR '${format}' ################### "
+              ./run_sdk_container -n "${vms_container}" -C "${image_image}" \
+                  -v "${vernum}" \
+                  ./image_to_vm.sh --format "${format}" --board="${arch}-usr" \
+                      --from "${CONTAINER_IMAGE_ROOT}/${arch}-usr/latest" \
+                      --image_compression_formats=bz2
+          done
+
+          # copy resulting images
+          ./run_sdk_container -n "${vms_container}" \
+              -v "${vernum}" \
+              cp --reflink=auto -R "${CONTAINER_IMAGE_ROOT}/${arch}-usr/" "./${images_out}"
```

```suggestion
              cp --reflink=auto -R "${CONTAINER_IMAGE_ROOT}/${arch}-usr/" "./${images_out}"
```
In case we have space problems on the runner, would it work to move here?

---

There should be enough space to run this workflow comfortably if we remove containers once we're done with them. If we need more space on the runners, we can always tweak the flavors.